### DISCUSSION TITLE
How to seed the storage locally

### DISCUSSION THREAD

**Discussion by groyGetaway** [supabase/supabase#12744] [Questions]
Hi, I'm trying to seed the storage for local development.

Figured out the storage info and in the db in the `storage.buckets` and `storage.object` tables. This means I can easily set these up in the seed.sql.
Last part missing is, where are the files that are used in the storage locally? Knowing this, I could add some files there with a script when the devs use the supabase cli.

If there is a better way to seed the storage for local dev, I'm all ears!

Edit : another option would be to be able to run javascript at the end of the `supabase db reset` process. As its easy to do all the above with the supabse storage javascript api

**Comment by GaryAustin1** [comment]
When running locally I believe the storage-api defaults to your local filesystem storage.
Here is the code that seems to handle it https://github.com/supabase/storage-api/blob/master/src/storage/backend/file.ts for what is worth.
